### PR TITLE
DATAENG-3469 linter changes

### PR DESCRIPTION
**Pull request by olga-savelyeva**

### CODE REVIEW

File: linter/rules/label_custom_check.py [hunk 1]
```diff
@@ -11,12 +11,16 @@ def applies_to() -> Tuple[str, ...]:
...
+            # Check if the remaining characters follow the rule
+            words_in_label = label.split()
+            return all(starts_with_capital_or_digit_or_special_char(word) for word in words_in_label)
+
         return True
 
+
     def message(self) -> str:
-        return 'Every word in label should start with capital letter, digit or special character'
+        return 'Every word in label should start with capital letter, digit, special character or whitespace'
```
Comment (olga-savelyeva) [inline linter/rules/label_custom_check.py:26]: why word should start with whitespace?

Comment (olga-savelyeva) [reply linter/rules/label_custom_check.py:26]: I guess better to highlight that label cannot be blank and should start with ...

File: linter/rules/label_custom_check.py [hunk 2]
```diff
@@ -11,12 +11,16 @@ def applies_to() -> Tuple[str, ...]:
     def run(self, lookml_object, runtime_params: Union[Any, None] = None) -> bool:
         label = lookml_object.get('label')
         if label:
-            words_in_label = label.split(' ')
-            for word in words_in_label:
-                valid = starts_with_capital_or_digit_or_special_char(word)
-                if not valid:
-                    return False
+            # Check if all characters are whitespaces
+            if all(char.isspace() for char in label):
```
Comment (olga-savelyeva) [inline linter/rules/label_custom_check.py:15]: .isspace() returns True also for new line char, but this is good, so not only whitespaces.

Comment (julia-knutelska-ordon) [reply linter/rules/label_custom_check.py:15]: True, left it and amended the comment. ✅

File: linter/rules/label_custom_check.py (hunk 1 above)
Comment (julia-knutelska-ordon) [reply linter/rules/label_custom_check.py]: Ah, thank you for catching that! Should be good now ✅

**Review by julia-knutelska-ordon — APPROVED** [review]

**Pull request merged**